### ISSUE TITLE
Auto repair

### ISSUE THREAD

**Issue by bhalevy** [scylladb/scylladb#17852]
Currently repairs are scheduled externally, via the ScyllaDB Manager.
With [incremental repair](https://github.com/scylladb/scylladb/issues/22472), repairs should be run much more regularly, multiple times a day.
Move repair scheduling into ScyllaDB core, so repairs can be scheduled internally, based on a user-provided configuration.

**Comment by bhalevy** [comment]
Cc @tgrabiec @asias

**Comment by denesb** [comment]
Tablet repair scheduler was added in https://github.com/scylladb/scylladb/pull/22176.

We now have the infrastructure to schedule repairs in core. What is missing is the scheduling part.

**Comment by mykaul** [comment]
@denesb - is the scheduling part going to 2025.2.0?

**Comment by denesb** [comment]
> [@denesb](https://github.com/denesb) - is the scheduling part going to 2025.2.0?

No, we didn't even start working on the design, pushing this out to 2025.3.

**Comment by mykaul** [comment]
@denesb - do we have an update for this?

**Comment by denesb** [comment]
> [@denesb](https://github.com/denesb) - do we have an update for this?

No, we didn't start on this yet.

**Comment by tgrabiec** [comment]
I think the description needs updating about what exactly is left to do, as there was some work done. For example:

> This infrastructure may also be used to implement a cluster-repair API that can distribute the tablet repair load jobs, throttle them properly, and synchronize with ongoing tablet-migrations on a single tablet resolution.

We have `nodetool cluster repair`.

**Comment by denesb** [comment]
Yes, as-is it is very unclear what this issue is about, we have implemented (or decided not to implement) most of the things it talks about.
This issue is supposed to be about automated repair, I will update the description accordingly.

**Comment by denesb** [comment]
Replaced by https://scylladb.atlassian.net/browse/SCYLLADB-98.
